### PR TITLE
feat: refactor settings and add TRiCAMlab env

### PR DESCRIPTION
**Pull request by mirestrepo**

### CODE REVIEW

File: src/trials/quizTrials.js [hunk 1]
```diff
@@ -17,27 +17,73 @@ const blueOrGreen = [
...
+    `${lang.quiz.rules.job}`,
+    `${lang.quiz.rules.blue_req_20}`,
+    `${lang.quiz.rules.green_vary_bonus}`,
+    `${lang.quiz.rules.twenty_five_secs_green}`,
+    `${lang.quiz.rules.bonus_green_spike}`,
+    `${lang.quiz.any_questions}`,
+    `${lang.quiz.retake}`,
+  ];
+}
+// const quizPrompts = [
```
Comment (mirestrepo) [inline src/trials/quizTrials.js:66]: Remove uncommented code. It will only make it more confusing in the future

File: src/trials/quizTrials.js [hunk 2]
```diff
@@ -190,7 +246,7 @@ const checkRetake = (blockSettings) => {
       const prevData = jsPsych.data.getLastTrialData().values()[0];
       const prevAnswers = prevData.answer;
 
-      const correctAnswers = [
+      let correctAnswers = [
```
Comment (mirestrepo) [inline src/trials/quizTrials.js:249]: since this variable is already defined, you shouldn't need the `let` again

File: src/trials/quizTrials.js [hunk 3]
```diff
@@ -199,6 +255,19 @@ const checkRetake = (blockSettings) => {
...
+      if(process.env.REACT_APP_settingsOverload === "remove-probability"){
+        correctAnswers = [
+          `${lang.quiz.answer_opts.green}`,  
+          `${lang.quiz.answer_opts.false}`,
+          `${lang.quiz.answer_opts.true}`,
+          `${lang.quiz.answer_opts.true}`,
+          `${lang.quiz.answer_opts.true}`,
+        ]
+      }
+
```
Comment (mirestrepo) [inline src/trials/quizTrials.js:268]: Remove these white spaces

**Review by mirestrepo — COMMENTED** [review]
I left few suggestions. Seems like I can't approve for some reason and only comment

**Review by anvo2 — APPROVED** [review]

**Pull request merged**